src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/DatabaseSchemaObject.java

```diff
@@ -0,0 +1,20 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver.executor;
+
+public class DatabaseSchemaObject extends SchemaObject {
+
+  public static final SchemaObjectType TYPE = SchemaObjectType.DATABASE;
+
+  public DatabaseSchemaObject() {
+    super(TYPE, SchemaObjectName.MISSING);
+  }
+
+  @Override
+  public VectorConfig vectorConfig() {
```

Comment:
I think it was mentioned earlier that these should be promoted up from `SchemaObject` to `CollectionSchemaObject`.

Reply:
Possible, follow up, skipping for now.